packages/langium/src/lsp/completion/completion-provider.ts

```diff
@@ -442,6 +441,11 @@ export class DefaultCompletionProvider implements CompletionProvider {
         });
     }
 
+    protected filterKeyword(_context: CompletionContext, keyword: ast.Keyword): boolean {
```

Comment:
The method name is ambiguous. In the JS context, "filter" is usually understood as include, not "filter out".